isotovideo

```diff
@@ -73,19 +74,24 @@ bmwqemu::clean_control_files();
 my $init = 1;
 
 my $cpid;
+my $backend;
 
 # all so ugly ...
 sub signalhandler {
 
-    # do not start a race about the results between the threads
-
     my $sig = shift;
-    diag("signalhandler $$: got $sig");
+    print STDERR "$$: signalhandler got $sig\n";
```

Comment:
this was just my debugging - diag is fine. And no, diag can't write in the system journal - the worker catches all output of isotovideo into autoinst-log.txt